docs/experimental-features/investigation-guide-actions.asciidoc

```diff
@@ -19,27 +19,59 @@ image::images/ig-timeline.png[Timeline with query pre-loaded from investigation
 
 NOTE: You can only create interactive investigation guides with custom rules because Elastic prebuilt rules can't be edited. However, you can duplicate a prebuilt rule, then configure the investigation guide for the duplicated rule.
 
-You can configure an interactive investigation guide when you <<rules-ui-create,create a new rule>> or <<edit-rules-settings,edit an existing rule>>. When configuring the rule's settings (the *About rule* step for a new rule, or the *About* tab for an existing rule), expand the *Advanced settings*, then scroll down to the *Investigation guide* field.
+You can configure an interactive investigation guide when you <<rules-ui-create,create a new rule>> or <<edit-rules-settings,edit an existing rule>>. 
 
+. When configuring the rule's settings (the *About rule* step for a new rule, or the *About* tab for an existing rule), expand the *Advanced settings*, then scroll down to the *Investigation guide* Markdown editor.
++
 [role="screenshot"]
 image::images/ig-investigation-guide-editor.png[Investigation guide editor field]
 
-Add the following syntax to the *Investigation guide* Markdown editor to configure a query. The syntax will appear as a clickable button in the investigation guide.
+. Place the editor cursor where you want to add the query button in the investigation guide, then select the Insights icon (image:images/ig-insights-icon.png[Insights icon,16,15]) in the toolbar. The *Add investigation query* builder form appears.
++
+[role="screenshot"]
+image::images/ig-investigation-query-builder.png[Add investigation guide UI,600]
+
+. Complete the query builder form to create an investigation query:
+.. *Label*: Enter the text to appear on the query button.
+.. *Description*: (Optional) Enter additional text to include with the button.
+.. *Filters*: Select fields, operators, and values to build the query. Click *OR* or *AND* to create multiple filters and define their relationships.
++
+To use a field value from the alert as a query parameter, enter the field name surrounded by double braces — such as `{{kibana.alert.example}}` — as a custom option for the filter value.
```

Comment:
QQ: should we call the double brackets "double braces" or "double curly brackets"? I'm using "double curly brackets" [here](https://github.com/elastic/security-docs/pull/3045/files#diff-3ad82a825ca079969d9356d9c29d31c7c3f364c4ca02a12f4a4c2bc28b50ba99R17) but am open to changing it if it should be "double braces".

Reply:
Good question @nastasha-solomon! From [Wikipedia](https://en.wikipedia.org/wiki/Bracket#Curly_brackets_or_braces) 😵‍💫:

> Curly brackets (British) or braces (American), { and }, are also known as "curly braces",[[30]](https://en.wikipedia.org/wiki/Bracket#cite_note-30) "definite brackets", "swirly brackets", "birdie brackets", "French brackets", "Scottish brackets", "squirrelly brackets",[[31]](https://en.wikipedia.org/wiki/Bracket#cite_note-31) "gullwings", "seagulls", "squiggly brackets", "twirly brackets", "Tuborg brackets" (DK), "accolades" (NL), "pointy brackets", "fancy brackets", "M Braces", "moustache brackets", "squiggly parentheses", or "flower brackets" (India).

I originally went with "double braces" since that's what's in the UI help text, but "double curly brackets" seems more descriptive. Unless we all agree on "squirrelly brackets" instead! 🐿

Reply:
I'm partial to "fancy brackets" 🎩 or "gullwings" 🐦, but "double curly brackets" works too ;)